cupy/cuda/stream.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, null=False, non_blocking=False):
     def __del__(self):
         if self.ptr:
             runtime.streamDestroy(self.ptr)
-            self.ptr = None
+            self.ptr = 0
```

Comment:
Do we have to set `self.ptr` to 0 explicitly even if `Stream` object itself will be deleted at the same time?